pom.xml

```diff
@@ -476,7 +476,7 @@
 				<plugin>
 					<groupId>com.google.code.sortpom</groupId>
 					<artifactId>maven-sortpom-plugin</artifactId>
-					<version>2.3.0</version>
+					<version>2.4.0</version>
```

Comment:
I know. At first there was a sorting issue that forced me to upgrade. If I create another pull request, you might need to accept it to make this one pass